t4.cpp

```diff
@@ -1,4 +1,6 @@
 namespace {
 intt;
 int x = 1.1;
```

Comment:
variable 'x' is non-const and globally accessible, consider making it const [cppcoreguidelines-avoid-non-const-global-variables]